### ISSUE TITLE
[CUSTOM REQUEST] Custom Whiteboard Display

### ISSUE THREAD

**Issue by bullseye555** [rollerderby/scoreboard#541]
This one's for me - but I don't have any other way to contact you @frank-weinberg !

Had a game over the weekend where I was PBM and my PBT's were relatively new and weren't keeping up with paperwork (let's not even go there....) but my ePLT's were ontop of everything

As a PBM, we've run a 'hot box' in the past where a whiteboard would be in the box and have the skaters numbers on 5/6/7 penalties. It would be nice-to-have a Whiteboard screen where the skaters are sorted by number of penalties, rather than their skater number (and as a stretch-goal, only when they're on a minimum of 5 penalties) - I had a look at the whiteboard screen coding, and it's a bit beyond me to do this myself and was hoping I could engage you to do this as a custom screen (or add it as a feature in 5.x if you feel if would be useful globally)

Understand if you don't have the time, or it's not as simple to code - but thought I'd reach out and ask :)

Thanks,

~Bullseye

**Comment by frank-weinberg** [comment]
First thought: I would never use this myself. I want the entries to remain stationary so I can quickly locate each skater. The colors are enough to draw attention where needed.
But: I would not have a problem with it being an option (or 2 options) that each user of the screen can set.

Without looking at the code: Hiding all entries with fewer than X penalties should be quite easy to do. We already determine the count and thus could based on it toggle a css-class that sets display:none. The hardest part would be adding the interface for setting the option.
Changing the sorting should also be doable but a lot more work.

Also: This is the perfect place for such a request. I am not the only person in the world who could do this and by putting it here there's a chance that someone else can pick it up when I don't have time (as is currently the case).

**Comment by bullseye555** [comment]
Thanks Frank - happy to wait on this one. My HTML is rusty as %%^*, and CSS was only a couple of years old when I learnt HTML so it wasn't yet part of the curriculum... (I know they're easy enough to learn, but it's all that thing we don't have... time!)

**Comment by atkinsonm** [comment]
I'm curious how the penalty box would use this. In my experience it's typically the PT/PLTs' responsibilities to track how many penalties skaters have; as a PBT I keep the numbers too (not the penalty codes) but just for extra checks and balances (though without the penalty codes it's limited). As long as the PTs/PW are keeping up, I would think PBTs could do without tracking the number of penalties per skater.

I do think that a custom screen could be useful but for other purposes. For example, a bench coach might want a read-only screen similar to the roster screen but with penalties. That would save bench coaches coming up to the head ref, PLTs, or scorekeeping table (I couldn't tell you how many people have asked me and I've redirected them to PLTs) asking about how many penalties Skater XYZ has.

It seems like that screen already exists. There's a penalty whiteboard display from the list of displays on the index page.

![image](https://github.com/rollerderby/scoreboard/assets/3342243/d3e19d43-486c-4421-bcbe-a53cd1ef6947)

**Comment by bullseye555** [comment]
@atkinsonm
You're correct - the Penalty Whiteboard screen does exist and shows both teams. We regularly have screens at team benches that displays this for the purpose you mentioned 

This request is now over a year old, and I've not had the need/want for this screen as I'd thought at the time of the request 

Based on that, I'll close this as not planned.

**Comment by bullseye555** [comment]
Selecting the right option this time...
